src/ui/Facet/FacetSettings.ts

```diff
@@ -189,6 +185,37 @@ export class FacetSettings extends FacetSort {
     return this.settingsButton;
   }
 
+  private buildSettingsButton() {
+    this.settingsButton = $$('div', { className: 'coveo-facet-header-settings' }).el;
+    this.settingsButton.innerHTML = SVGIcons.icons.more;
+    SVGDom.addClassToSVGInContainer(this.settingsButton, 'coveo-facet-settings-more-svg');
+
+    this.hideElementOnMouseEnterLeave(this.settingsButton);
+    this.makeSettingsButtonAccessible();
+  }
+
+  private hideElementOnMouseEnterLeave(el: HTMLElement) {
+    const mouseLeave = () => (this.closeTimeout = window.setTimeout(() => this.close(), 300));
```

Comment:
Is the `window` necessary?

Reply:
Indeed it is ... without it, typescript throws an type error and I cannot cast my way out of it. I think the nodejs typedefinitions are to blame. I also see `window.setTimeout` used in multiple files. I think trying to clean this is better left for a separate PR.